index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Flutter is a mobile app SDK for iOS and Android
+title: Welcome to Flutter
```

Comment:
Oh, I thought this was a stop-gap change, waiting for the real page to come.
Tell you what...after this PR if people who come to the website are still confused about what Flutter is, we can definitely change it back.